pkg/blob/nodeserver.go

```diff
@@ -164,11 +165,23 @@ func (d *Driver) mountBlobfuseWithProxy(args string, authEnv []string) (string,
 	return output, err
 }
 
-func (d *Driver) mountBlobfuseInsideDriver(args string, authEnv []string) (string, error) {
+func (d *Driver) mountBlobfuseInsideDriver(args string, protocol string, authEnv []string) (string, error) {
+	var cmd *exec.Cmd
+
 	klog.V(2).Infof("mounting blobfuse inside driver")
-	cmd := exec.Command("blobfuse", strings.Split(args, " ")...)
+	if protocol == Fuse2 {
+		klog.V(2).Infof("using blobfuse V2 to mount")
```

Comment:
I could not find `using blobfuse V2 to mount` logging in https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_blob-csi-driver/788/pull-blob-csi-driver-e2e/1595706774915125248/build-log.txt, is there sth. wrong with the blobfuse v2 test case or this PR?

Reply:
there is only `using blobfuse V1 to mount` logging

Reply:
nvm, I think it's related to driver restart test which make driver logs missing after restart.